Project/Source/Components/ComponentAudioListener.cpp

```diff
@@ -72,3 +72,9 @@ void ComponentAudioListener::UpdateAudioListener() {
 	alListenerfv(AL_POSITION, position.ptr());
 	alListenerfv(AL_ORIENTATION, orientation);
 }
+
+void ComponentAudioListener::SetAudioVolume(float volume) {
+	gain = volume;
+	alListenerf(AL_GAIN, (ALfloat) volume);
+	UpdateAudioListener();

```

Comment:
Don't need it since you're updating only the gain of the Listener, not the position neither orientation